perf.groovy

```diff
@@ -1,49 +0,0 @@
-// Groovy Script: http://www.groovy-lang.org/syntax.html
```

Comment:
Note this file is explicitly called out in the netci configuration. Not sure if we have to delete it first?

Reply:
https://github.com/dotnet/dotnet-ci/pull/994